### PR TITLE
Fix post build signing props and enable post build signing.

### PR DESCRIPTION
**Pull request by jkoritzinsky**
Add back post-build specific signing items to Signing.props and enable PostBuildSign in the official build.

### CODE REVIEW

**Review by mmitche — APPROVED** [review]

**Comment by jkoritzinsky** [comment]
Failure looks to be https://github.com/dotnet/runtime/issues/44060

**Review by ViktorHofer — COMMENTED** [review]
Hooray 👍

**Comment by ViktorHofer** [comment]
image

from https://dnceng.visualstudio.com/internal/_build/results?buildId=893146&view=results

File: eng/Signing.props [hunk 1]
```diff
@@ -44,6 +44,15 @@
     <FileExtensionSignInfo Include=".pkg" CertificateName="8003" />
     <FileExtensionSignInfo Include=".deb;.rpm" CertificateName="LinuxSign" />
   </ItemGroup>
+  <ItemGroup Condition="'$(PrepareArtifacts) == 'true' and '$(PostBuildSign)' == 'true'">
```
Comment (ViktorHofer) [inline eng/Signing.props:47]: Missing the ' after $(PrepareArtifacts)'

Comment (ViktorHofer) [reply eng/Signing.props:47]: https://github.com/dotnet/runtime/pull/44971

**Pull request merged**